.github/workflows/test.yml

```diff
@@ -49,14 +49,6 @@ jobs:
     - if: ${{ matrix.os != 'windows-latest' }}
       run: python -m pip install --prefer-binary -v .[test]
```

Comment:
Typo? Same 2 lines repeated?

Reply:
Not really. It is 2 different runs.  They run the same command but in Windows case CMAKE_GENERATOR is defined.